package_manifest.yaml

```diff
@@ -65,6 +65,12 @@ copr_projects:
             - "{{ foreman_staging }}/rhel-{{ rhel_8 }}-x86_64"
           comps_file: "{{ inventory_dir }}/comps/comps-foreman-el{{ rhel_8 }}.xml"
           buildroot_packages: "{{ core_buildroot_packages }}"
+        - name: "rhel-{{ rhel_9 }}-x86_64"
+          external_repos:
+            - "https://yum.puppet.com/puppet7/el/{{ rhel_9 }}/x86_64/"
+            #- "{{ foreman_staging }}/rhel-{{ rhel_9 }}-x86_64"
+          comps_file: "{{ inventory_dir }}/comps/comps-foreman-el{{ rhel_9 }}.xml"
+          #buildroot_packages: "{{ core_buildroot_packages }}"
```

Comment:
there is no `foreman-build` for el9 yet.

Reply:
Does this break builds if we define the package to be included and it does not exist?

Reply:
Yes, mock freaks out it can't install things and aborts.